Firestore/core/src/remote/grpc_connection.cc

```diff
@@ -32,10 +32,14 @@
 #include "Firestore/core/src/util/log.h"
 #include "Firestore/core/src/util/statusor.h"
 #include "Firestore/core/src/util/string_format.h"
+#include "Firestore/core/src/util/warnings.h"
 #include "absl/memory/memory.h"
 #include "absl/strings/str_cat.h"
 #include "grpcpp/create_channel.h"
+
+SUPPRESS_DOCUMENTATION_WARNINGS_BEGIN()
 #include "grpcpp/grpcpp.h"
```

Comment:
Might as well just surround all the grpcpp includes in the block.

Reply:
Done.